index.html

```diff
@@ -548,73 +551,92 @@ <h2>
         MAY support content codings defined in previous versions of the draft for compatibility
         reasons.
       </p>
+      <h3>
+        `subscribe()` method
+      </h3>
       <p>
-        The <dfn>subscribe</dfn> method when invoked MUST run the following steps:
+        The <dfn>subscribe()</dfn> method when invoked MUST run the following steps:
       </p>
-      <ol>
-        <li>Let |promise| be <a>a new promise</a>.
+      <ol class="algorithm">
+        <li>If the user agent requires |options|'s {{PushSubscriptionOptionsInit/userVisibleOnly}}
+        member to be `true`, return [=a promise rejected with=] an {{"NotAllowedError"}}
+        {{DOMException}}.
```

Comment:
This would always reject if userVisibleOnly had to be true, rather than just when it is false.

```suggestion
        <li>If the |options| argument has a {{PushSubscriptionOptionsInit/userVisibleOnly}} value set to `false` and 
        the user agent requires it to be `true`, return [=a promise rejected with=] an {{"NotAllowedError"}}
        {{DOMException}}.
```

I'm not 100% on the bit where we this leans on defaults, your next step seems to do that though.